projectEuler3.py

```diff
@@ -0,0 +1,26 @@
+#Problem 3
+#The prime factors of 13195 are 5, 7, 13 and 29.
+#What is the largest prime factor of the number 600851475143 ?
+def main():
+    number = int(raw_input('Please Enter an Integer: '))
+    factor = 2
+    
+    def isPrime(x):
+        global largest
+        for i in range(2,int(x ** (0.5) + 1)):
+            if x%i == 0:
+                return False
+        return True    
+        
+    if isPrime(number) == True:
+        largest = number
+    else:    
+        while number > 1:
+            if number % factor == 0 and isPrime(number / factor) == True:
+                largest = number/factor
```

Comment:
space between '/'
usually if you do the same operation more than once, you should create a variable to hold it. (this is up to you, since its only twice here)